src/ProjectTemplates/ComponentsWebAssembly.ProjectTemplates/content/ComponentsWebAssembly-CSharp/Client/Program.cs

```diff
@@ -19,6 +19,15 @@ public static async Task Main(string[] args)
             builder.RootComponents.Add<App>("app");
 
             // use builder.Services to configure application services.
+			builder.Services.AddSingleton<HttpClient>(s =>
```

Comment:
Fix indentation

Reply:
Indentation fixed back to spaces.

Reply:
Also found the project and another test I missed.